lib/PuppeteerSharp.Tests/DevToolsTracingTests/DevToolsTracingTests.cs

```diff
@@ -53,11 +53,11 @@ public override async Task DisposeAsync()
             }
         }
 
-        [PuppeteerTest("tracing.spec.ts", "Tracing", "should output a trace")]
+        [PuppeteerTest("devtoolstracing.spec.ts", "Tracing", "should output a trace")]
```

Comment:
This `spec.ts` should match the location of the test in puppeteer. I think we shouldn't change this.